src/styles/card.scss

```diff
@@ -0,0 +1,94 @@
+$font-color-primary: #060b35;
+$font-color-collateral: #616070;
+$background-color:#fff;
+$border-color: #f3f3f3;
+$button-color:#00acdc;
+$hug: 16px;
+
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  background-color: $background-color;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  padding: $hug;
+
+  &__image {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin-top: 16px;
+    margin-left: 3px;
+  }
+
+  &__title {
+    display: block;
+    color: $font-color-primary;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-top: 40px;
+  }
```

Comment:
1. You use `<h2>` and it has default margins, the task says that you should reset them
<img width="273" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/49764881/7d645679-6cb0-4f08-adad-125e3b53ecdd">

2. `display: block;` is default

Reply:
👍